app/components/djs/search.hbs

```diff
@@ -0,0 +1,9 @@
+<PowerSelectMultiple
+  @options={{this.badges}}
+  @selected={{this.selectedBadges}}
+  @allowClear={{true}}
+  @placeholder="Select some badges..."
```

Comment:
Maybe `Filter by badge` ? And let's get this in i18n

Reply:
k boss should be good now